packages/stems/src/assets/styles/shadows.css

```diff
@@ -0,0 +1,5 @@
+:root, root {
```

Comment:
nice, i notcited we have this for the tile element in web, is this something we can reuse, or not the same for this?

Reply:
Yeah, the idea is going forward you should be able to just do `var(--near)` for a shadow anywhere in stems or web and it'll pick this up.